python_modules/libraries/dagster-dg/dagster_dg/scaffold.py

```diff
@@ -111,6 +120,31 @@ def scaffold_project(
         if populate_cache:
             RemoteComponentRegistry.from_dg_context(cl_dg_context)  # Populate the cache
 
+    # Update pyproject.toml
+    if cl_dg_context.is_workspace:
+        entry = {
+            "path": str(cl_dg_context.root_path.relative_to(cl_dg_context.workspace_root_path)),
+        }
+
+        with modify_toml(dg_context.pyproject_toml_path) as toml:
+            if not has_toml_node(toml, ("tool", "dg", "workspace", "projects")):
+                projects = tomlkit.aot()
+                set_toml_node(toml, ("tool", "dg", "workspace", "projects"), projects)
+                item = tomlkit.table()
+            else:
+                projects = get_toml_node(
+                    toml,
+                    ("tool", "dg", "workspace", "projects"),
+                    (tomlkit.items.AoT, tomlkit.items.Array),
```

Comment:
what an unfortunate acronym

Reply:
lol yes don't know why they didn't just spell it out